dwds/lib/src/debugging/location.dart

```diff
@@ -43,6 +43,9 @@ class Location {
     return Location._(JsLocation.fromZeroBased(scriptId, jsLine, jsColumn),
         DartLocation.fromZeroBased(dartUri, dartLine, dartColumn));
   }
+
+  @override
+  String toString() => '${dartLocation.toString()} -> ${jsLocation.toString()}';
```

Comment:
No need to explicitly call `toString` here (string interopolation does that by default)

```suggestion
  String toString() => '${dartLocation()} -> $jsLocation';
```